debug-bar-action-and-filters-addon.php

```diff
@@ -204,6 +208,7 @@ function debug_bar_action_and_filters_addon_display_filters() {
 						$table .= '<li>[<em>' . esc_html__( 'object', 'debug-bar-actions-and-filters-addon' ) . '</em>] ' . $signature . '</li>';
 					} else {
 						// Type 8 - undetermined
+						$signature = uniqid( 'undetermined_callback_' );

```

Comment:
No need for this as there is a default at the top of the if/else which handles this. Also, the `uniqid()` will probably lead to unjustified inflation of the numbers.